intellij-templates/intyg-common.xml

```diff
@@ -115,15 +115,6 @@
       <option name="JAVA_DECLARATION" value="true" />
     </context>
   </template>
-  <template name="validationmindate" value="@org.junit.jupiter.api.Nested&#10;class IncludeValidationMinDateTest extends se.inera.intyg.common.support.facade.testsetup.model.validation.ValidationMinDateTest {&#10;&#10;    @java.lang.Override&#10;    protected se.inera.intyg.common.support.facade.model.CertificateDataElement getElement() {&#10;        return $elementImpl$.toCertificate(null, 0, textProvider);&#10;    }&#10;&#10;    @java.lang.Override&#10;    protected int getValidationIndex() {&#10;        return $validationIndex$;&#10;    }&#10;&#10;    @java.lang.Override&#10;    protected java.lang.String getId() {&#10;        return $id$;&#10;    }&#10;&#10;    @java.lang.Override&#10;    protected java.time.LocalDate getMinDate() {&#10;        return $minDate$;&#10;    }&#10;}" description="create text-block validation min date" toReformat="true" toShortenFQNames="true">
```

Comment:
@mww-sopra-steria I'm unsure if I'm supposed to change this file or if this is something that generates by itself?

Reply:
These are the live templates we added to easier setup the tests for mindate validation.

Its my understanding that we no longer uses this LiveTemplate for both Min/Max date validation when we set dates, so this could be considered redundant and we should be able to remove both templates.